### PR TITLE
Force version of dataclasses to .6

### PR DESCRIPTION
**Pull request by shanaxel42**

### CODE REVIEW

**Comment by ttung** [comment]
Can you verify as part of your test plan that python 3.7 + dataclasses doesn't have any averse effects?

File: REQUIREMENTS.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # installed, and bombs after the fact.
 python-dateutil < 2.8.1
 click
-dataclasses;python_version<"3.7"
+dataclasses;python_version==.6
```
Comment (ttung) [inline REQUIREMENTS.txt]: shouldn't this just be `dataclasses == 0.6`?

Comment (shanaxel42) [reply REQUIREMENTS.txt]: yuuuup

**Comment by shanaxel42** [comment]
> Can you verify as part of your test plan that python 3.7 + dataclasses doesn't have any averse effects?

I'm currently using `3.7.3` on my computer and things are chill, do you want me to add a test?

File: REQUIREMENTS.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # installed, and bombs after the fact.
 python-dateutil < 2.8.1
 click
-dataclasses;python_version<"3.7"
+dataclasses=0.6
```
Comment (ttung) [inline REQUIREMENTS.txt]: i think this needs to be a double equal.

**Comment by ttung** [comment]
> I'm currently using 3.7.3 on my computer and things are chill, do you want me to add a test?

nope, just wanted a quick sanity check.

**Pull request merged**